index/scorch/scorch.go

```diff
@@ -503,3 +515,15 @@ func (s *Scorch) unmarkIneligibleForRemoval(filename string) {
 func init() {
 	registry.RegisterIndexType(Name, NewScorch)
 }
+
+func parseToInteger(v interface{}) (int, error) {
+	switch v.(type) {
+	case float64:
+		return int(v.(float64)), nil
```

Comment:
Go has syntax to allow remembering the newly typed value in a type assertion, so you don't have to do the assertion a second time when returning the value here.  See this example: https://tour.golang.org/methods/16

Reply:
done!